bigtop-packages/src/common/ranger/install_ranger.sh

```diff
@@ -146,7 +146,7 @@ else
     COMPONENT_LIB_DIR=${PREFIX}/${RANGER_COMPONENT_DIR}/tomcat/webapps/kylin/WEB-INF/lib
```

Comment:
Could you remove Ranger related changes from this PR? It should be done in separate JIRA/PR.